applications/processor/http-request-processor/src/main/resources/META-INF/dataflow-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1,2 @@
+configuration-properties.classes=org.springframework.cloud.stream.app.processor.http.request.HttpRequestProcessorProperties,\
+  org.springframework.cloud.stream.app.processor.http.request.HttpRequestProcessorProperties$Retry
```

Comment:
Looks like there is no `Retry` class any more.

Reply:
Yes. We discussed that. It no longer applies here. The binder already implements Retry.

Reply:
So, are you going to fix this then?

Reply:
```suggestion
```